### PR TITLE
add a permalink button to copy cost url with params / daterange

### PR DESCRIPTION
**Pull request by wdvr**

### CODE REVIEW

**Comment by huydhn** [comment]
You could take a look at this existing component https://github.com/pytorch/test-infra/blob/main/torchci/components/CopyLink.tsx and see if it can be reused instead

File: torchci/pages/cost_analysis.tsx [hunk 1]
```diff
@@ -576,6 +594,17 @@ export default function Page() {
             <FaInfoCircle />
           </Typography>
         </Tooltip>
+        <Tooltip title="Copy Permalink">
```
Comment (ZainRizvi) [inline torchci/pages/cost_analysis.tsx:597]: There's a copy link component already.  Can you use that instead?

See:

https://github.com/pytorch/test-infra/blob/main/torchci/components/CopyLink.tsx

Comment (wdvr) [reply torchci/pages/cost_analysis.tsx]: updated

**Comment by wdvr** [comment]
switched to the existing componenet @huydhn @ZainRizvi

**Pull request merged**